crates/iceberg/src/io/file_io.rs

```diff
@@ -43,10 +43,21 @@ use crate::{Error, ErrorKind, Result};
 /// | GCS                | `storage-gcs`     | `gcs`       |
 #[derive(Clone, Debug)]
 pub struct FileIO {
+    scheme: String,
```

Comment:
> Instead of storing them, how about we make `FileIOBuilder` clone and serializable? This could avoid exposing internals and reuse everything there.

Hi, there is no way for us to get a `FileIOBuilder` from existing `Table` instance.

Reply:
Oh, I got your idea: you mean store `FileIOBuilder` directly instead `(String, HashMap<String, String>)`. Seems nice, I will take a try.